developer/docs/arvo/subscriptions.md

```diff
@@ -312,8 +319,10 @@ Otherwise, if our input is `%off` and we're already subscribed (i.e. `val`
 is false), then we unsubscribe from `:source` and set `val` back to true (`&`), 
 again using our handy inverted cell constructor mold `:_`:
 
-    :_	+>.$(val &)
-    ~[[ost.bow %pull /subscribe [our.bow %source] ~]]
+```
+    :_  +>.$(val &)                                     ::  28
+    ~[[ost.bow %pull /subscribe [our.bow %source] ~]]   ::  29
```

Comment:
Maybe we should change this to [foo bar baz]~ like the other apps for consistency.